src/messages/MMgrConfigure.h

```diff
@@ -32,19 +33,30 @@ class MMgrConfigure : public Message
   // Default 0 means if unspecified will include all stats
   uint32_t stats_threshold = 0;
 
+  bool osd_perf_metric_query = false;
```

Comment:
The bool was just a placeholder until the list was added, right?

Reply:
yes (or whatever the query data structure ends up being)

Reply:
I suppose yes, we can drop it.
Checking the size of the query list might be enough to tell if there are queries to process or not.